test/sslapitest.c

```diff
@@ -12130,7 +12250,8 @@ int setup_tests(void)
     ADD_ALL_TESTS(test_large_app_data, 28);
     ADD_TEST(test_cleanse_plaintext);
 #ifndef OPENSSL_NO_OCSP
-    ADD_TEST(test_tlsext_status_type);
+    ADD_TEST(test_tlsext_status_type_single);
+    ADD_TEST(test_tlsext_status_type_multi);
```

Comment:
This test is not yet convincing for multiple responses being stapled.
Add an explicit check for the number of responses actually included.